x/staking/src/abci_handler.rs

```diff
@@ -53,9 +53,11 @@ impl<
     }
 
     fn init_genesis<DB: Database>(&self, ctx: &mut InitContext<'_, DB, SK>, genesis: GenesisState) {
-        self.keeper
+        // TODO
+        let _updates = self
+            .keeper
             .init_genesis(ctx, genesis)
-            .expect("Cannot perform genesis actions.")
+            .expect("Cannot perform genesis actions.");
```

Comment:
When you use Result::expect(), could you explain in the message why you think it's safe to do so? https://doc.rust-lang.org/std/result/enum.Result.html#recommended-message-style

Reply:
@joneskm  Thank you for the feedback. 

I added detailed explanation for each `expect` and updated some handlers.  Please, check it. If you'll notice any unclear explanation, add a comment, please,